client/components/info-tooltip/style.scss

```diff
@@ -2,3 +2,33 @@
 	cursor: help;
 	color: $gray-dark;
 }
+
+&.tooltip.popover.info-tooltip__container {
+	.popover__inner {
+		background: $gray-dark;
+	}
+
+	.popover__arrow {
+		border-top-color: $gray-dark;
+	}
+
+	.info-tooltip__contents {
+		padding: 4px;
+		color: $white;
+
+		h1, h2, h3 {
+			margin: 0 0 8px 0;
+			color: $white;
+			font-weight: 600;
+			font-size: 1.3em;
```

Comment:
why were `font-weight: 600;` and `font-weight: 600;` added here from `info-tooltip__contents`?